app/controllers/state_file/questions/nc_county_controller.rb

```diff
@@ -0,0 +1,11 @@
+module StateFile
+  class Questions::NcCountyController < Questions::QuestionsController
+    include ReturnToReviewConcern
+    before_action :set_ivars
+
+    def set_ivars
```

Comment:
@powersurge360 What is ivars?

Reply:
Rubyist [shorthand for instance variables](https://docs.ruby-lang.org/en/master/contributing/glossary_md.html#:~:text=Instance%20Variable.%20Refers%20to%20a%20Ruby%20instance%20variable%20like%20%E2%80%98@foo%60).